yew/src/utils.rs

```diff
@@ -105,3 +106,121 @@ impl<IN, OUT> IntoIterator for NodeSeq<IN, OUT> {
         self.0.into_iter()
     }
 }
+
+/// Reference to either a &'static str or an owned String
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum StringRef {
```

Comment:
Would this be better achieved using a [`Cow`](https://doc.rust-lang.org/std/borrow/enum.Cow.html)?

Reply:
There was more reason to use a separate type in the earlier versions of this PR. Yes, I can replace it with `Cow` to reduce duplication.

Reply:
This makes conversions from various string types harder and introduces more breaking changes to the VTag API, so I'll keep `StringRef` as is.

Reply:
@bakape what about new-typing `Cow`?

Reply:
That might trim the code a bit. I'll try and see.

Reply:
No, does not make thing any DRYer.

Reply:
@bakape hmm that's unfortunate. Can you please remind me why `StringRef` is needed? It seems like API methods could take `Into<Cow<'static, str>>` rather than `Into<StringRef>`

Reply:
@jstarry There used to be more reasons. Swapped StringRef for Cow.